helm-chart/sefaria-project/templates/rollout/nginx.yaml

```diff
@@ -90,6 +90,10 @@ spec:
             value: "varnish-{{ .Values.deployEnv }}-{{ .Release.Revision }}"
           - name: SEARCH_HOST
             value: "{{ .Values.nginx.SEARCH_HOST }}"
+          {{- if .Values.linker.enabled }}
+          - name: LINKER_HOST
+            value: "varnish-{{ .Values.deployEnv }}-{{ .Release.Revision }}"
```

Comment:
I admit I don't really understand this. But should this say `varnish`? Maybe you meant `linker`?

Reply:
I did - copy-paste error :/